Automattic/wp-calypso
# A4A > Marketplace: Add Pressable content for the new hosting page

Closes https://github.com/Automattic/jetpack-genesis/issues/437
Closes https://github.com/Automattic/jetpack-genesis/issues/439
Closes https://github.com/Automattic/jetpack-genesis/issues/451

## Proposed Changes

This PR:

- Adds Pressable hosting additional features
- Adds Pressable hosting features
- Adds Pressable hosting testimonials 
- Creates a new component to move the common hosting benefits and reuse them on both WPs.com and the Pressable hosting page.
- Updates the testimonial images (using PNG)
- Updates the features section icon sizes to match the design


## Why are these changes being made?

* For the new hosting page

## Testing Instructions

* Open the A4A live link
* Go to Marketplace > Hosting > Click the Premier Agency Hosting tab > Verify the UI looks as shown below:

![screencapture-agencies-localhost-3000-marketplace-hosting-2024-07-26-11_08_44](https://github.com/user-attachments/assets/446c3005-b600-4a7d-9339-89bd0a188b5c)


## Pre-merge Checklist
